HerPortal/appsettings.json

```diff
@@ -3,41 +3,42 @@
         "AppBaseUrl": "https://localhost:5001",
         "ReferralReminderCrontab": "0 7 * * *"
     },
-    
+
     "ConnectionStrings": {
-        "PostgreSQLConnection": "UserId=herportaldev;Password=herportaldev;Server=localhost;Port=5432;Database=herportaldev;Integrated Security=true;Include Error Detail=true;Pooling=true"
+        "PostgreSQLConnection": "UserId=postgres;Password=postgres;Server=localhost;Port=5432;Database=herportaldev;Integrated Security=true;Include Error Detail=true;Pooling=true"
```

Comment:
The mismatch in these between website and portal had been driving me up the wall slightly...!

Reply:
This is my fault! I'm using `herdev` as the username and password on the public site, I just am very careful about not committing that change 😂